jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/skins/AudioPlayerSkin.kt

```diff
@@ -0,0 +1,124 @@
+package org.wycliffeassociates.otter.jvm.controls.skins
+
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import io.reactivex.Observable
+import io.reactivex.disposables.Disposable
+import javafx.application.Platform
+import javafx.fxml.FXML
+import javafx.fxml.FXMLLoader
+import javafx.scene.Node
+import javafx.scene.control.Button
+import javafx.scene.control.SkinBase
+import javafx.scene.control.Slider
+import javafx.scene.layout.HBox
+import org.kordamp.ikonli.javafx.FontIcon
+import org.wycliffeassociates.otter.common.device.AudioPlayerEvent
+import org.wycliffeassociates.otter.jvm.controls.AudioPlayerNode
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import java.util.concurrent.TimeUnit
+
+class AudioPlayerSkin(private val player: AudioPlayerNode) : SkinBase<AudioPlayerNode>(player) {
```

Comment:
This skin and SourceAudioSkin are very similar. I would make it one file with one fxml where particular items are shown or hidden depending on whether it's a source audio or simple audio.